db/datamigrate/create_log_entries_from_data.rb

```diff
@@ -0,0 +1,27 @@
+class CreateLogEntriesFromData
```

Comment:
We'll need to execute this datamigration ASAP after deploying; logs will not show data in the web UI until we do.